### PR TITLE
Add docs for Ohme integration

### PR DESCRIPTION
**Pull request by dan-r**
## Proposed change

Adds docs for my Ohme integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: home-assistant/core#132574
- Link to parent pull request in the Brands repository: home-assistant/brands#5022
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced comprehensive documentation for the Ohme integration with Home Assistant.
	- Added configuration details for connecting various Ohme EV chargers, including Ohme Home Pro, Ohme Home, Ohme Go, and Ohme ePod.
	- Included a configuration flow with required fields for user login (Email and Password).
	- Listed sensors provided by the integration, detailing their functionalities, including current status and power metrics.
	- Specified button functionality for approving charging when the status is pending approval.

### CODE REVIEW

File: source/_integrations/ohme.markdown [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  - Sensor
+  - Car
+ha_release: 2025.1
+ha_iot_class: Cloud Polling
+ha_codeowners:
+  - '@dan-r'
+ha_config_flow: true
+ha_domain: ohme
+ha_platforms:
+  - binary_sensor
```
Comment (zweckj) [inline source/_integrations/ohme.markdown]:
```suggestion
  - sensor
```

File: source/_integrations/ohme.markdown [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+---
+
+The **Ohme** {% term integration %} allows you to connect your Ohme EV charger to Home Assistant.
```
Comment (zweckj) [inline source/_integrations/ohme.markdown]:
```suggestion
The **Ohme** {% term integration %} allows you to connect your [Ohme EV](https://ohme-ev.com/) charger to Home Assistant.
```

File: source/_integrations/ohme.markdown [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+## Removing the integration
+
+This integration follows standard integration removal. No extra steps are required.
+
+{% include integrations/remove_device_service.md %}
+
+## Sensor
+The {% term integration %} provides the following sensors:
+
+- Status - Current status of the charger. Possible states: Unplugged, Pending approval, Plugged in, Charging
```
Comment (zweckj) [inline source/_integrations/ohme.markdown]:
```suggestion
- **Status** - Current status of the charger. Possible states: `Unplugged`, `Pending approval`, `Plugged in`, `Charging`
```
make UI strings bold (also below), states ticked

**Review by zweckj — DISMISSED** [review]

File: source/_integrations/ohme.markdown [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+## Third party accounts
+If you signed up to Ohme with a third party account like Google, you will need to [reset your password](https://api.ohme.io/fleet/index.html#/authentication/forgotten-password) before configuring this integration.
+
+
+## Removing the integration
+
+This integration follows standard integration removal. No extra steps are required.
+
+{% include integrations/remove_device_service.md %}
```
Comment (frenck) [inline source/_integrations/ohme.markdown]: Let's move this to the bottom of the documentation

File: source/_integrations/ohme.markdown [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    required: true
+    type: string
+Password:
+    description: "Password to log in to your Ohme account."
+    required: true
+    type: string
+{% endconfiguration_basic %}
+
+## Third party accounts
+If you signed up to Ohme with a third party account like Google, you will need to [reset your password](https://api.ohme.io/fleet/index.html#/authentication/forgotten-password) before configuring this integration.
```
Comment (frenck) [inline source/_integrations/ohme.markdown]: Isn't this a pre-requisite?

File: source/_integrations/ohme.markdown [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+The **Ohme** {% term integration %} allows you to connect your [Ohme](https://ohme-ev.com/) EV charger to Home Assistant.
+
+
+## Prerequisites
+
+- A compatible charger. This integration has been tested with:
+  - Ohme Home Pro
+  - Ohme Home
+  - Ohme Go
+  - Ohme ePod
```
Comment (frenck) [inline source/_integrations/ohme.markdown]: This should be a supported device section. For reference:

https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/docs-supported-devices

File: source/_integrations/ohme.markdown [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+{% include integrations/config_flow.md %}
+{% configuration_basic %}
+Email:
+    description: "Email to log in to your Ohme account."
+    required: true
+    type: string
```
Comment (frenck) [inline source/_integrations/ohme.markdown]: These things don't work for configuration_basic and are meant for YAML-configuration (which this is not)

```suggestion
```

File: source/_integrations/ohme.markdown [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+Password:
+    description: "Password to log in to your Ohme account."
+    required: true
+    type: string
```
Comment (frenck) [inline source/_integrations/ohme.markdown]:
```suggestion
```

File: source/_integrations/ohme.markdown [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+- **Power**
+- **Current**
+- **Energy**
+- **CT current** - If a current transformer (CT) was installed with your charger, this will show the current used by your whole home
+
+## Button
+
+The {% term integration %} provides the following button:
+
+- **Approve charge** - If sensor **Status** is `Pending approval`, this will approve the charge
```
Comment (frenck) [inline source/_integrations/ohme.markdown]: Let's format these as shown in the developer documentation:

https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/docs-supported-functions

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @dan-r 👍 


../Frenck

**Pull request merged**